src/cli/ProjectsCLIController.ts

```diff
@@ -4,32 +4,38 @@ import { BaseCLIController } from './BaseCLIController'
 import { getRepoConfig } from '../utils'
 
 export type Project = {
-  _id: string
-  name: string
+  _id?: string
+  name?: string
   key: string
 }
 
 export class ProjectsCLIController extends BaseCLIController {
+  public async getActiveProject() {
+    const stateProject = StateManager.getFolderState(this.folder.name, KEYS.PROJECT_ID)
+    if (stateProject) {
+      return stateProject
+    }
+    const repoConfig = await getRepoConfig(this.folder)
```

Comment:
It's possible that the repo config is not defined or doesn't contain a project, would this handle that case?

Reply:
it does not. I think if the repo config does not exist then the project in the user config should be used. That's guranteed to be set since we force the user to set a project on login